make/package.cmake

```diff
@@ -44,6 +44,9 @@
 cmake_minimum_required(VERSION 3.7)
 set(CTEST_SOURCE_DIRECTORY "${CTEST_SCRIPT_DIRECTORY}/..")
 
+# ensure that we check glibc for version incompatibilities.
+SET(BUILD_PACKAGE ON CACHE BOOL "build DynamoRIO for packaging purposes by performing glibc checks")
```

Comment:
This actually won't set it in the builds done by the ctest_build() step we use: it needs to be inside base_cache which is set down at line 162 or so.

It would be nice to have a check that would catch this failing to be set: but I don't know that there's a simple way to do it b/c how would we know we're doing a build that wants the check.